types/riviere.d.ts

```diff
@@ -1,5 +1,6 @@
 export function riviere(options?: {
   color?: boolean,
+  styles: Array<string>,
```

Comment:
Can you also update the README with what is the defaullt and what is this?

Reply:
Done